src/Confluent.SchemaRegistry/CachedSchemaRegistryClient.cs

```diff
@@ -13,30 +13,54 @@
 // limitations under the License.
 //
 // Refer to LICENSE for more information.
+#pragma warning disable CS0618
+#pragma warning disable CS0612
 
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using System.Linq;
 using System;
 using System.Threading;
+using Confluent.Kafka;
 
 
 namespace Confluent.SchemaRegistry
 {
     /// <summary>
     ///     A caching Schema Registry client.
+    ///
+    ///     The following method calls cache results:
```

Comment:
`CachedSchemaRegistry` (following the java client) is a pretty bad abstraction, and there was potential at this point to do something better, but the value didn't seem worth the effort and I backed out of it. Instead, here's slightly better docs.